translations/en-us.yml

```diff
@@ -1921,12 +1921,12 @@ osf-components:
             view_files_detail: 'Click a file name to go to view the file in the OSF. Opens file in a new tab.'
             upload: Upload
             upload_detail_first: 'There are two ways to upload files. Open the storage provider or folder where you intend to upload files; you can then drag files from your desktop into files list area OR click the green plus symbol' 
-            upload_detail_second: ', then click “Upload Files”, and select your files.'
+            upload_detail_second: ', then click “Upload Files”, and select your files. (Note: File names with special characters, like <code>&</code>, <code>%</code>, and <code>/</code> may cause unexpected behavior).'
```

Comment:
I would suggest not goading users who may be so inclined. More neutral phrasing or simple guidance not to use those characters is better. 'Note: Using file names with ...are not recommended.' may work. Also using 'such as' instead of 'like' is more professional. I would also remove the parenthesis as they are redundant with 'Note:'.